src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -282,22 +305,34 @@ class adminLogs extends viewModelBase {
             this.pauseLogs();
             return;
         }
-        
-        data = data.trim();
-        
-        if (!this.headerSeen) {
-            this.headerSeen = true;
-            return;
+
+        let customizedMsg = false;
+        if (this.connectionJustOpened()) {
```

Comment:
can we somehow avoid this logic here? andmove to to "after connected callback"?

Reply:
Leaving this as is
We are replacing the _'header msg'_ that is received the first time